sdk/cognitivelanguage/ai-language-conversations/samples-dev/sample_analyze_orchestration_app_conv_response.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to analyze user query using an orchestration project.
+ * In this sample, orchestration project's top intent will map to a conversation project.
+ *
+ * @summary Orchestration project with conversational response
+ * @azsdk-weight 50
+ */
+
+import { ConversationAnalysisClient, ConversationalTask } from "@azure/ai-language-conversations"
+import { AzureKeyCredential } from "@azure/core-auth";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+//Get secrets
+//You will have to change these environment variables for the sample to work
```

Comment:
The customer should set them, right?
```suggestion
//You will have to set these environment variables for the sample to work
```

Reply:
Well yes, the customer should set them, by "changing" I meant changing the dummy environment variables in the samples.env file, but "setting" makes more sense anyway.